app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["wife"]].husband = Person.people[person["name"]]
```

Comment:
There is a potential issue here: if `person['wife']` is not in `Person.people`, this line will raise a `KeyError`. Ensure that the referenced person exists in `Person.people` before accessing it.